Model/PageInterface.php

```diff
@@ -249,14 +249,14 @@ public function addChildren(PageInterface $children);
     public function getChildren();
 
     /**
-     * Add blocs.
```

Comment:
yes, but i can't change the method name, without breaking BC

so lets do this in another PR, because ```addBlockS``` should be ```addBlock```....

this is symfony generating stuff :(